plugin/code_actions.py

```diff
@@ -340,6 +373,17 @@ def show_bulb(self) -> None:
     def hide_bulb(self) -> None:
         self.view.erase_regions('lsp_bulb')
 
+    def show_annotations(self) -> None:
```

Comment:
Things in the gutter are hoverable. Look at GitGutter for example. Perhaps we can use that to hover over this lightbulb and show code actions?

I was thinking to use annotations for diagnostics -- and remove the "active diagnostic" in the status bar to unclutter it. Not all servers use code actions, so reserving that feature just for code actions seems like a waste.

Reply:
> Perhaps we can use that to hover over this lightbulb and show code actions?

So how would the flow look like?
 - The bulb is hovered, which shows a minihtml popup with "3 Code actions"? And then clicking it reveals the native popup?
 - Or on hovering the bulb we show a minihtml popup with already populated lists of code actions?

> I was thinking to use annotations for diagnostics

I guess it's a separate discussion but I think that might not be very practical as there is limited space (at least for the annotations added by `add_regions`.

Reply:
Ah, the horizontal spacing is limited? Then I suppose it's suitable only for telling the user there exist code actions.

But if we have the annotations show up, perhaps we can remove the bulb? It would otherwise be double information.

Reply:
I would use the gutter icon for everything if it would be possible to make gutter icon clickable. There is a way to handle that using some hack (https://github.com/daveleroy/sublime_debugger/issues/68) but it's not cool (we don't really know the position or extend of the icon reliably).

So I'm up for using those annotations, removing the bulb and the `show_code_actions_bulb` setting.